atom/renderer/renderer_client_base.cc

```diff
@@ -136,9 +136,7 @@ void RendererClientBase::RenderThreadStarted() {
 #if defined(OS_MACOSX)
   base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
   bool scroll_bounce = command_line->HasSwitch(switches::kScrollBounce);
-  base::ScopedCFTypeRef<CFStringRef> rubber_banding_key(
-      base::SysUTF8ToCFStringRef("NSScrollViewRubberbanding"));
-  CFPreferencesSetAppValue(rubber_banding_key,
+  CFPreferencesSetAppValue(CFSTR("NSScrollViewRubberbanding"),
```

Comment:
@zcbenz are you ok with this change?

Reply:
I'm not familiar with this, but from docs of [CFSTR](https://developer.apple.com/documentation/corefoundation/cfstr?language=objc) this looks fine.